ansible_collections/arista/avd/roles/eos_config_deploy_cvp/tasks/v1/present.yml

```diff
@@ -3,26 +3,26 @@
 - name: "Collecting facts from CVP {{ inventory_hostname }}."
   tags: [always]
   arista.cvp.cv_facts:
-  register: CVP_FACTS
+  register: cvp_facts
```

Comment:
should this be reverted back as well?

Reply:
let's merge the other one first and rebase this one